.github/workflows/create-pr.yml

```diff
@@ -0,0 +1,83 @@
+name: Create PR with external dependency
+
+on:
+  repository_dispatch:
+    types: [create_pr]
+
+jobs:
+  build:
+    name: Create PR with external dependency
+    runs-on: ubuntu-latest
+    steps:
+      - name: Verify Event and set env
+        env:
+          BUILD_URL: ${{ github.event.client_payload.build_url }}
+          PR_BRANCH: ${{ github.event.client_payload.branch }}
+          PACKAGE: ${{ github.event.client_payload.dependency_package }}
+        run: |
+          echo "PR_BRANCH=$PR_BRANCH" >> $GITHUB_ENV
+          echo "BUILD_URL=$BUILD_URL" >> $GITHUB_ENV
+          echo "PACKAGE=$PACKAGE" >> $GITHUB_ENV
+
+          PULL_REQUEST_GIT_USER='${{ vars.PULL_REQUEST_GIT_USER }}'
+          echo "GIT_USER_NAME=$(echo "$PULL_REQUEST_GIT_USER" | jq -r .name)" >> $GITHUB_ENV
+          echo "GIT_USER_EMAIL=$(echo "$PULL_REQUEST_GIT_USER" | jq -r .email)" >> $GITHUB_ENV
+
+          BUILD_URL_IS_ACCEPTED=$(echo '${{ vars.ACCEPTED_BUILD_URL_PREFIXES }}' | jq --arg url "${{ env.BUILD_URL }}" 'any(.[]; . as $prefix | $url | contains($prefix))')
+          echo "BUILD_URL_IS_ACCEPTED=$BUILD_URL_IS_ACCEPTED" >> $GITHUB_ENV
```

Comment:
I have been experimenting a bit. It seems like that it is not enough to make sure an error is shown. In order to skip an entire workflow you need to set exit code 1.
So that is what I have been doing. It cleans up the code a bit.